src/solids4FoamModels/solidModels/vertexCentredNonLinGeomTotalLagPressureDisplacementSolid/vertexCentredNonLinGeomTotalLagPressureDisplacementSolid.C

```diff
@@ -924,125 +924,125 @@ vertexCentredNonLinGeomTotalLagPressureDisplacementSolid::geometricStiffnessFiel
 }
 
 
-//Foam::tmp<tensorField>
-//vertexCentredNonLinGeomTotalLagPressureDisplacementSolid::pBarSensitivityField
-//(
-//    const pointTensorField pGradDRef
-//) const
-//{
-//    // Prepare tmp field
-//    tmp<tensorField> tresult
-//    (
-//        new tensorField(mesh().nPoints(), Foam::tensor::zero)
-//    );
-//#ifdef OPENFOAM_NOT_EXTEND
-//    tensorField& result = tresult.ref();
-//#else
-//    tensorField& result = tresult();
-//#endif
+Foam::tmp<tensorField>
+vertexCentredNonLinGeomTotalLagPressureDisplacementSolid::pBarSensitivityField
+(
+    const pointTensorField pGradDRef
+) const
+{
+    // Prepare tmp field
+    tmp<tensorField> tresult
+    (
+        new tensorField(mesh().nPoints(), Foam::tensor::zero)
+    );
+#ifdef OPENFOAM_NOT_EXTEND
+    tensorField& result = tresult.ref();
+#else
+    tensorField& result = tresult();
+#endif
 
-//    //Calculate the bulk modulus
-//    const scalar E = 70e9;
-//    const scalar nu = 0.3;
-//    const scalar K( (nu*E/((1.0 + nu)*(1.0 - 2.0*nu))) + (2.0/3.0)*(E/(2.0*(1.0 + nu))) );
+    //Calculate the bulk modulus
```

Comment:
When hard-coding things like this for testing, it is good practice to print out a warning so we don't forget about it, e.g.
```
Warning
    << "E/nu/K hard-coded in pBarSensitivityField for testing!" << endl;
```